charts/karpenter/templates/clusterrole-core.yaml

```diff
@@ -45,6 +45,13 @@ rules:
   - apiGroups: ["admissionregistration.k8s.io"]
     resources: ["validatingwebhookconfigurations", "mutatingwebhookconfigurations"]
     verbs: ["get", "watch", "list"]
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
+    verbs: ["get", "watch", "list"]
```

Comment:
Is get/watch/list necessary for conversion webhooks? or is there a subset we don't need?

Reply:
This is for the conversion webhooks